coderedcms/templates/coderedcms/pages/base.html

```diff
@@ -6,6 +6,9 @@
   <head>
         {% block tracking %}
         {% if settings.coderedcms.AnalyticsSettings.ga_tracking_id %}
+            {% if settings.coderedcms.AnalyticsSettings.ga_track_button_clicks %}
+                <meta name="tracking:click" value="True" />
```

Comment:
I would probably replace this custom meta tag with a variable in the script below where google tag manager is pulled in:
```javascript
<script>
  tracking_click = true;
  window.dataLayer = window.dataLayer || [];
  ...
</script>
```
Then if the variable is undefined or false you know not to track the onclick event.